java/developer-guide/knowledge-base/technical-articles/converting-outlook-message-file-msg-to-tiff-image/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Converting Outlook Message File (MSG) to TIFF Image
+type: docs
+weight: 130
+url: /java/converting-outlook-message-file-msg-to-tiff-image/
+---
+
+
+Convert an Outlook MSG file to a TIFF image is not supported in Aspose.Email for Java.
```

Comment:
Wrap this line with {{% alert color="primary" %}} and {{% /alert %}} shortcodes.